frontend/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -57,19 +52,18 @@ class PlotlyChart extends React.PureComponent<PropsWithHeight> {
     return <iframe title="Plotly" src={url} style={{ width, height }} />
   }
 
-  public generateSpec = (figure: ImmutableMap<string, any>): any => {
-    const el = this.props.element
+  private isFullScreen = (): boolean => !!this.props.height
```

Comment:
why is this a function?

Reply:
This is only for code readability purposes